layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,45 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode f dash-functional
+                      (lean-mode :location (recipe
+                                            :fetcher github
+                                            :repo leanprover/lean
+                                            :files ("src/emacs/*.el")))))
+(defun lean/init-f ()
+  (use-package f
+    :defer t))
```

Comment:
Just defining init function for package is enough instead of using `use-package` with `:defer t` in case you aren't configuring it. Same goes for dash-functional and mmm-mode.

So having just `(defun lean/init-f ())` instead of above should be enough.